CODEOWNERS

```diff
@@ -28,10 +28,7 @@
 # is not properly picked up in Github.
 * @cilium/janitors
 /.github/ @cilium/contributing
-/.github/workflows/ @cilium/ci-structure
-/.github/workflows/coccicheck.yaml @cilium/ci-structure @cilium/bpf
-/.github/workflows/docs.yaml @cilium/ci-structure @cilium/docs-structure
-/.github/workflows/images.yaml @cilium/ci-structure @cilium/build
+/.github/workflows/ @cilium/maintainers
```

Comment:
Why are these other codeowners removed? We can still assign specific files to specific codeowners, but the overall `.github/workflows` be managed by `cilium/maintainers` (which you already have right now)

Reply:
we can't have specific files to be assigned to different teams, all of the .github workflows need to be seem by maintainers.